### PR TITLE
[MIG] sale_order_line_date: Migration to 17.0

### PR DESCRIPTION
**Pull request by yannoliv**

### CODE REVIEW

**Comment by Anxo82** [comment]
LGTM :+1:

File: sale_order_line_date/models/sale_order.py [hunk 1]
```diff
@@ -14,7 +14,9 @@ class SaleOrder(models.Model):
 
     @api.onchange("commitment_date")
     def _onchange_commitment_date(self):
-        """Update empty commitment date order lines with commitment date from sale order"""
+        """
+        Update empty commitment date order lines with commitment date from sale order
+        """
```
Comment (ChristianSantamaria) [inline sale_order_line_date/models/sale_order.py:19]: This change should be into "pre-commit commit" -> [[IMP] sale_order_line_date: pre-commit auto fixes](https://github.com/OCA/sale-workflow/pull/3091/commits/17e716ba9930a05a2f0f032fa29457075f849618)

**Review by ChristianSantamaria — CHANGES_REQUESTED** [review]
Functional review :+1: 

Only a simple change in commits

**Review by Anxo82 — APPROVED** [review]

File: sale_order_line_date/models/sale_order_line.py [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+
+    def write(self, vals):
+        res = super().write(vals)
+        moves_to_upd = set()
+        if "commitment_date" in vals:
+            for move in self.move_ids:
+                if move.state not in ["cancel", "done"]:
+                    moves_to_upd.add(move.id)
+        if moves_to_upd:
+            self.env["stock.move"].browse(moves_to_upd).write(
```
Comment (BenjaHe) [inline sale_order_line_date/models/sale_order_line.py:40]: Bug if vals.get("commitment_date") is False 

This case occurs when user set the commitment_date and remove it after the delivery is generated

**Comment by DorianMAG** [comment]
Hi @yannoliv 
Any chance to update your PR.
Runboat is blocked

**Comment by DorianMAG** [comment]
Possible duplicate migration 
https://github.com/OCA/sale-workflow/pull/2962

**Comment by HaraldPanten** [comment]
Closing, as contributors are working on https://github.com/OCA/sale-workflow/pull/2962

**Pull request closed without merging**